test/README.md

```diff
@@ -1,72 +1,38 @@
+1. Install the ["Remote Development"](https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.vscode-remote-extensionpack) VS Code Extension
+   - Restart VSCode
+2. You will be prompted that there is a Dev Container detected, press "reopen in a container"
+   - If no prompt, press F1 and enter "Remote-Containers: Open Folder in Container.."
+3. Install [Testify](https://marketplace.visualstudio.com/items?itemName=felixjb.testify) VS Code Extension
+4. Go to the test file you want to run and click "Run Test" above the name.
+
+> _NOTE: VScode Extensions you have already installed won't automatically installed on the dev container you are now running with now.
+You will need install them again for this environment if you need them._
+
+## VSCode Debug a specific test
+_Tested up to VSCode Version: `1.54.2`_
 
-## VSCode Testing Current File
-1. To test the current file, make sure you have set up the following run configuration in the `.vscode/launch.json` file:
-   - Tested up to VSCode Version: `1.46.1`
-      - (later versions seem to have broken debugging)
-```
-{
-    // Use IntelliSense to learn about possible attributes.
-    // Hover to view descriptions of existing attributes.
-    // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
-    "version": "0.2.0",
-    "configurations": [
-        {
-            "name": "Transpile & Run",
-            "preLaunchTask": "transpile",
-            "runtimeExecutable": "${workspaceFolder}/node_modules/.bin/ava",
-            "runtimeArgs": [
-                "debug",
-                "--break",
-                "--verbose",
-                "--color",
-                "--serial",
-                "--watch=false",
-                "build/ts-to-es6/test/**/${fileBasenameNoExtension}.js",
-            ],
-            "request": "launch",
-            "skipFiles": [
-                "<node_internals>/**"
-            ],
-            "type": "node",
-            "outputCapture": "std",
-            "port": 9229,
-            "sourceMaps": true,
-            "trace": true,
-            "protocol": "auto",
-            "console": "integratedTerminal"
-        },
-        {
-            "name": "Run",
-            "runtimeExecutable": "${workspaceFolder}/node_modules/.bin/ava",
-            "runtimeArgs": [
-                "debug",
-                "--break",
-                "--verbose",
-                "--color",
-                "--watch=false",
-                "--fail-fast=false",
-                "build/ts-to-es6/test/**/${fileBasenameNoExtension}.js",
-            ],
-            "request": "launch",
-            "skipFiles": [
-                "<node_internals>/**"
-            ],
-            "type": "node",
-            "outputCapture": "std",
-            "port": 9229,
-            "sourceMaps": true,
-            "trace": true,
-            "protocol": "auto",
-            "console": "integratedTerminal"
-        },
-    ]
-}
-```
```

Comment:
Why is this removed? The default way to run tests is using test launch files.

Perhaps add a general overview section where we explain that we are using a different environment via these plugins.